daemon/src/taker_cfd.rs

```diff
@@ -191,6 +191,13 @@ where
         // to take it more than once
         self.current_maker_offers.replace(maker_offers);
 
+        // Cleanup own order feed
+        // Due to the 1:1 relationship between order and cfd we can never create another cfd for the
+        // same order id.
+        self.projection_actor
+            .send(projection::Update(self.current_maker_offers))
+            .await?;
```

Comment:
Somehow I struggled to understand the commit message together with the change. What you've done is remove the offer taken by the user even if the daemon decides that it cannot go through with taking it. Right? Maybe I can open a follow-up PR that makes these lines a bit clearer.